pydash/numerical.py

```diff
@@ -383,6 +386,43 @@ def std_deviation(array):
     return math.sqrt(variance(array))
 
 
+def call_math_operator(value1, value2, callback):
+    """Return the result of the math operation on the given values.
+
+    Args:
+        value1 (int/float/str): Value passed by the user.
+        value2 (int/float/str): Value passed by the user.
+        callback (mixed, optional): Callback applied on the given values.
+
+    Returns:
+        float: Result after the math operation is conducted on the given
+            values.
+
+    Example:
+
+        >>> call_math_operator(5, 4, lambda x, y: x - y)
+        1.0
+        >>> call_math_operator(4, 6, lambda x, y: x * y)
+        24.0
+        >>> call_math_operator('5', '10', lambda x, y: x + y)
+        15.0
+        >>> call_math_operator('25', '5', lambda x, y: x / y)
+        5.0
+
+    .. versionadded:: TODO
+    """
+    # If the user has given any Nonetype values, we assign 0 as the default
+    # value to the values.
+    value1 = 0 if not value1 else value1
+    value2 = 0 if not value2 else value2
```

Comment:
The default value should be an argument to this function since `0` won't always be valid (e.g. we shouldn't use `0` for division or multiplication since for division we might get a division-by-zero error and for multiplication we'd get `0` as the return value). Instead, for division/multiplication we'd probably want the default to be `1`.